specs/BrowserProcessExited.md

```diff
@@ -19,7 +19,7 @@ process is active, and moving to a new WebView2 Runtime version after a
 `NewBrowserVersionAvailable` event.
 
 This event is raised for both expected and unexpected termination of the
-collection of processes, after all resources, including the user data folder, 
+collection of processes, after all resources, including the user data folder,
```

Comment:
Do we have a diagram somewhere that shows the processes?

Reply:
Number one search hit for "Webview2 process" is [Process Model](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model) which has diagrams.

Reply:
Oh, thanks, that was a good idea. There's also a link to that page in this spec :(

![image](https://user-images.githubusercontent.com/18429489/116729625-3ab70f80-a99c-11eb-8ba2-44dce3d72e4d.png)

So `BrowserProcessExited` is raised when everything goes away, `ProcessFailed` is raised when any process fails (including the Browser process).  `BrowserProcessExited` isn't on the WebView object alongside `ProcessFailed`, because the Environment object is fundamentally associated with the Browser.

Makes sense, thanks.

Reply:
The WebView2 Runtime has the same process model as Edge/Chromium. We document that here: [Process model](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model). The "browser process" is in charge of all other processes. The "BrowserProcess" part in `BrowserProcessExited` refers to that "browser process" in charge, and this is the PID being provided.

 `ProcessFailed` indicates failure of one process of any kind (browser, render, GPU, etc.). `BrowserProcessExited` indicates exit of the "browser process" by crash or normal exit. The "browser process" is last to exit most of the times, but there are some cases in which some descendants might outlive it; the `BrowserProcessExited` event will only be raised upon those have exited too. This is the reason "collection of processes" was introduced during the [earlier review](https://github.com/MicrosoftEdge/WebView2Feedback/pull/714#discussion_r540536781).

I'll try to make this clearer in the documentation.

Reply:
Sounds good. Please update documentation.

Reply:
Updated.